### ISSUE TITLE
changing the scale from 1. to 0.3 didn't make any difference in the size of the object in hand

### ISSUE THREAD

**Issue by monacv** [otaheri/GrabNet#4]
So I changed the scale from 1. which was working great for the camera object to 0.3 (meaning 30cm?) for the wine bottle. In either case of 1. or 0.3 for the value assigned to scale, unfortunately, the wine bottle is the same exact small (inappropriately scaled) size in the hand. Could you please guide how to fix this?

Also, is there a way I could find the exact scale for the objects? Is there a 3D model repo that has such infos?

Thank you.
![Screenshot from 2021-04-13 18-59-35](https://user-images.githubusercontent.com/76495162/114631582-dd784a00-9c8a-11eb-9e32-b896dd35e408.png)

**Comment by otaheri** [comment]
From your previous issue, I noticed that the wine bottle was too large and the code automatically down-scaled it by a factor of ~370. Is it the same object or did you change its scale? 
![image](https://user-images.githubusercontent.com/19238978/114675219-4baf2200-9d08-11eb-9734-eacf9c19737f.png)

The hand scale is as a real hand and the dimensions of the meshes should be in meters. It means that your wine bottle's height should approximately be 0.3 m. If it is not, you should accordingly up-scale or down-scale so that it is realistic. The scale value could be more than 1 in case you didn't know, so please try with a higher number if the object is too small.

Also if you want to prevent auto-scaling of the code, you can comment out the following line in the code in Colab.
https://github.com/otaheri/GrabNet/blob/0b428a72d9575e196503ec56b4c4f5b4fde65914/grabnet/tests/demo_colab.py#L157
I hope this helps.

**Comment by monacv** [comment]
Thanks a lot for your response. So I commented that line and ran it an it doesn't show a hand grabbing it when scale was 1.0 

![Screenshot from 2021-04-15 18-39-56](https://user-images.githubusercontent.com/76495162/114946961-37f3e080-9e1a-11eb-9dc0-6a73f63ac086.png)

when I ran again with scale 0.1 this is what I get. So how can I make it such that it looks like a realistically looking hand grabbing a wine bottle?
![Screenshot from 2021-04-15 18-40-52](https://user-images.githubusercontent.com/76495162/114947015-5063fb00-9e1a-11eb-93ce-923e10ed0c4a.png)
